portfolio.html

```diff
@@ -8,6 +8,284 @@
     <link href="portfolio.css" rel="stylesheet">
 </head>
 <body>
+
+    <!-- Background images first section -->
+    <div class="background-images-desktop">
+        <img src="./images/desktop_version/frame_headline/Frame.png" class="desktop-frame" alt="Frame group of icon">
+        <img src="./images/desktop_version/frame_headline/Group-38.png" class="desktop-group-38" alt="Group 38 image">
+        <img src="./images/desktop_version/frame_headline/Group2.png" class="desktop-group-2" alt="Group 2 image">
+        <img src="./images/desktop_version/frame_headline/Frame-big.png" class="desktop-frame-big" alt="Frame big icon">
+        <img src="./images/desktop_version/frame_headline/Group-37.png" class="desktop-group-37" alt="Group 37 image">
+        <img src="./images/desktop_version/frame_headline/Frame-middle.png" class="desktop-frame-middle" alt="Frame middle icon">
+        <img src="./images/desktop_version/frame_headline/Frame-right.png" class="desktop-frame-right" alt="Frame right icon">
+        <img src="./images/desktop_version/frame_headline/Group-35.png" class="desktop-group-35" alt="Group 35 image">
+    </div>
+
+    <!-- Desktop version header -->
+    <header class="header">
+        <ul class="menu">
+            <li class="menu menu-items1"><a href="#"></a>Portfolio</li>
+            <li class="menu menu-items2"><a href="#"></a>About</li>
+            <li class="menu menu-items3"><a href="#"></a>Contact</li>
+        </ul>
+    </header>
+
+    <!-- Desktop version headline -->
+    <section class="desktop-headline">
+        <div class="desktop-headline-first-text">
+            <h2 class="desktop-headline-heading">Hey there. I’m Norbi<br><span>I’m a software developer</span></h2>
```

Comment:
- [ ] This section is being rendered using position absolute, this is not recommended, please use only flex or grid
![image](https://user-images.githubusercontent.com/62705947/186935835-452c5c88-f43c-4814-bf4d-c5f0bb2c35af.png)